clients/stellar-relay-lib/src/connection/connector/connector.rs

```diff
@@ -86,18 +87,12 @@ impl Connector {
 		body: &[u8],
 	) -> Result<(), Error> {
 		let remote_info = self.remote_info.as_ref().ok_or(Error::NoRemoteInfo)?;
-		log::trace!(
+		trace!(
 			"verify_auth(): remote sequence: {}, auth message sequence: {}",
 			remote_info.sequence(),
 			auth_msg.sequence
 		);
 
-		let auth_msg_xdr = auth_msg.to_base64_xdr();
```

Comment:
I decided to remove this because the xdr was huge and may not be helpful at all.